data/itemGenerator.js

```diff
@@ -119,7 +124,7 @@ function generateOffer(age, baseUrl, modified, golden, free) {
 
 function generateImages(golden) {
   var out = [];
-  var imageCount = faker.random.number(golden ? {min: 4, max: 6} : 6);
+  var imageCount = faker.random.number(golden ? {min: 4, max: 6} : { max: 6 });
```

Comment:
My error checker gets confused when function call signatures are mixed with a single call (i.e. `number(max: number)` vs `number({ max: number, min: number })`).

So I've updated it to both use the same signature (with the `min`/`max` object)